internal/handlers/telegram/hander_test.go

```diff
@@ -0,0 +1,57 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"github.com/kyokomi/emoji"
+	"github.com/ritlug/teleirc/internal"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestStickerSmile(t *testing.T) {
+	correct := "<test>😄"
+	updateObj := tgbotapi.Update{
+		Message: &tgbotapi.Message{
+			From: &tgbotapi.User{
+				UserName: "test",
```

Comment:
Side note, but the `UserName` field is not required for Telegram users, only the `Name` field. 
To get around that, use the `user.String()` method which grabs the username if present, or the name otherwise.

Reply:
According to how the messages handler is forming the formatted message, UserName was used directly. Where can I find the `user` object?
```
formatted := tg.Settings.Prefix + u.Message.From.UserName +
		tg.Settings.Suffix + u.Message.Text
```

Reply:
nvm, I figured out what you meant

Reply:
fixed